doc/releases/release-notes-3.7.rst

```diff
@@ -60,6 +60,50 @@ Deprecated in this release
    Application developer will now need to set the advertised name themselves by updating the advertising data
    or the scan response data.
 
+ * POSIX API
+
+  * Deprecated :c:macro:`PTHREAD_BARRIER_DEFINE` has been removed.
```

Comment:
isn't this stuff that should go into the migration guide?

Reply:
I could technically add a script that would adjust .conf files automatically.

That would be a good thing for the migrations n guide

Reply:
Fixed!